sdk/storage/Azure.Storage.Blobs/tests/BlobClientTests.cs

```diff
@@ -541,6 +538,35 @@ async Task Verify(Task<Response<BlobContentInfo>> upload)
             }
         }
 
+        private static async Task DownloadAndAssert(long size, Stream stream, BlobClient blob)
+        {
+            var actual = new byte[Constants.DefaultBufferSize];
+            using var actualStream = new MemoryStream(actual);
+
+            // reset the stream before validating
+            stream.Position = 0;
```

Comment:
Use `Stream.Seek` for resetting

Reply:
Hmm, this seems more readable than 
`stream.Seek(0, SeekOrigin.Begin);`

since we are just setting an absolute position. Is there a reason that Seek is preferred?

Reply:
`Stream.Seek` has better perf for relative position changes.